tests/e2e/specs/customizer/header-builder/social-icons/social-icon-background-spacing.test.js

```diff
@@ -0,0 +1,30 @@
+import { createURL } from '@wordpress/e2e-test-utils';
+import { setCustomize } from '../../../../utils/customize';
+describe( 'Social Icons in the customizer', () => {
+	it( 'social icon background spacing should apply correctly', async () => {
+		const socialIconBackSpacing = {
+			'header-social-1-bg-space': '48px',
+			'header-desktop-items': {
+				primary: {
+					primary_center: {
+						0: 'social-icons-1',
+
+					},
+				},
+			},
+		};
+		await setCustomize( socialIconBackSpacing );
+
+		await page.goto( createURL( '/' ), {
+			waitUntil: 'networkidle0',
+		} );
+
+		await page.waitForSelector( '.ast-header-social-1-wrap .header-social-inner-wrap .ast-builder-social-element' );
```

Comment:
Please verify selectors:- https://share.bsf.io/4guKrXk7
```suggestion
		await page.waitForSelector( '.ast-header-social-1-wrap .ast-builder-social-element' );
```

Reply:
Done